pkg/apis/contrail/v1alpha1/manager_types.go

```diff
@@ -62,14 +62,34 @@ type VrouterManagerServiceConfiguration struct {
 	VrouterConfiguration `json:",inline"`
 }
 
-// KubemanagerService defines desired configuration of vRouter
+// ProvisionManagerService defines desired configuration of ProvisionManager
+// +k8s:openapi-gen=true
+type ProvisionManagerService struct {
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              ProvisionManagerServiceSpec `json:"spec,omitempty"`
+}
+
+// ProvisionManagerServiceSpec defines desired spec configuration of ProvisionManager
+// +k8s:openapi-gen=true
+type ProvisionManagerServiceSpec struct {
+	CommonConfiguration  PodConfiguration                            `json:"commonConfiguration,omitempty"`
+	ServiceConfiguration ProvisionmanagerManagerServiceConfiguration `json:"serviceConfiguration"`
+}
+
+// ProvisionmanagerManagerServiceConfiguration defines service configuration for Provisionmanager
+// +k8s:openapi-gen=true
+type ProvisionmanagerManagerServiceConfiguration struct {
```

Comment:
Why do we need this structure ProvisionmanagerManagerServiceConfiguration? Can't we just use ProvisionManagerConfiguration?

Reply:
To avoid changing manager's api. Otherwise configNodesInformation is added to manager's crd like it is in provision manager's crd.

Reply:
Hmm but it looks like ProvisionManagerConfiguration doesn't have configNodesInformation. ProvisionManagerServiceConfiguration has it.

Reply:
I was trying to do it like here https://github.com/Juniper/contrail-operator/pull/417/files#diff-6cca7d5c2a5f72f3552c59ec99e285070d61198fc3f38c48a26bd21fb22175e8R60 now I see that in this case it can be just 
```
ServiceConfiguration ProvisionManagerConfiguration `json:"serviceConfiguration"`
```

Reply:
I changed it to use ProvisionManagerConfiguration instead of ProvisionmanagerManagerServiceConfiguration